c2cgeoportal/scaffolds/create/project.yaml.in_tmpl

```diff
@@ -1,6 +1,6 @@
 project_folder: {{project}}
 project_package: {{package}}
 host: <host>
-checker_path: /${instance-id}/wsgi/check_collector?
+checker_path: /${instanceid}/wsgi/check_collector?
```

Comment:
It's better to be consistent and use "vars:instanceid" everywhere.
I see that https://github.com/camptocamp/c2cgeoportal/blob/1.5/c2cgeoportal/scaffolds/create/production.ini.in_tmpl#L13 use "instanceid" but "vars:instanceid" a few lines latter:
https://github.com/camptocamp/c2cgeoportal/blob/1.5/c2cgeoportal/scaffolds/create/production.ini.in_tmpl#L48
Ditto for development.ini.in_tmpl.
It would be great if you could make the changes here as well :)